cache/manager.go

```diff
@@ -615,6 +616,10 @@ func (cm *cacheManager) New(ctx context.Context, s ImmutableRef, sess session.Gr
 		}); rerr != nil {
 			return nil, rerr
 		}
+	} else if cm.Snapshotter.Name() == "overlaybd" && parent != nil {
+		rwLabels := make(map[string]string)
+		rwLabels["containerd.io/snapshot/overlaybd.writable"] = "dev"
```

Comment:
What’s “dev”?

Reply:
This is a type of upper layer in overlaybd config that can specify by label, maybe we should provide a const value in overlaybd code.

Reply:
When the "containerd.io/snapshot/overlaybd.writable" has been set as "dev", Snapshotter will create a R/W block device directly as rootfs without overlayfs // _local directory (upper) + RO overlaybd (lower)_
Just like devmapper :D

https://github.com/containerd/accelerated-container-image/blob/6dff475f080557cf8a8b5564a29ddee118258f56/pkg/snapshot/overlay.go#LL909C4-L909C4

Reply:
Thanks. Could you add that explanation in the code comment lines?